activate

```diff
@@ -72,15 +76,16 @@ serial="${serial%"${serial##*[![:space:]]}"}"
 live=false
 
 # report in
-if ! out=$(wget -O - -q "$SCRIPT?image=$image&release=$release&vendor=$vendor&product=$product&serial=$serial&live=$live"); then
-    echo "wget failed"
+URL="${SCRIPT}?image=${image}&release=${release}&vendor=${vendor}&product=${product}&serial=${serial}&live=${live}"
+if ! out=$(wget -O - -q ${URL}); then
```

Comment:
This is a bit hard to understand and I'm not 100% sure it does what you want as you are doing the condition based on the variable assignment success afaict.